godot-fmt/benches/gdext_tests.rs

```diff
@@ -0,0 +1,34 @@
+use std::{path::PathBuf, str::FromStr};
+
+use criterion::{black_box, criterion_group, criterion_main, Criterion};
+use godot_fmt::format_tokens;
+use proc_macro2::TokenStream;
+
+pub fn criterion_benchmark(c: &mut Criterion) {
+    // this little dance is needed because `cargo bench` and `cargo criterion`
+    // set the working directory slightly differently 🫠
+    let current_path = PathBuf::from(file!());
+    let test_cases_dir = current_path
+        .parent()
+        .unwrap()
+        .parent()
+        .unwrap()
+        .join("test-cases");

```

Comment:
Unfortunately it doesn't seem to work for me with `cargo bench` on Windows 10:
```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value:
   Os { code: 3, kind: NotFound, message: "Cannot find specified path." }',
   godot-fmt\benches\gdext_tests.rs:18:4
```
(OS message translated)

I used `env!("CARGO_MANIFEST_DIR")` elsewhere, which seemed to work quite reliably. Maybe look for that in existing code and use it here too?